### ISSUE TITLE
Can't set up 3 way dimmers in hassio

### ISSUE THREAD

**Issue by szbbalint** [rospogrigio/localtuya#630]
I have the device ID-s and local key but i got "Failed to authenticate with device. Verify that device id and local key are correct." error when i try to add a device in hassio in the integration.
I have 4 pieces martin jerry 3way dimmer and none of them work. 
I have  tried to readd the devices in the tuya app but it did not helped.
maybe the ID or the key is wrong? 
ID-s like: bf73ac694f36a4c3cfxdxn
keys like: ab5f3569b4526c5c
I get the keys from the iot.tuya site from the API explorer
What could be the problem?

**Comment by Scope666** [comment]
If it's the M&J STD01 like I have, the device is on Protocol 3.2 which localtuya doesn't support.  (Only 3.1 and 3.3)

Also I believe the STD01 has a chip that can't be flashed.     :(

**Comment by szbbalint** [comment]
Yep theye are STD01 ones. I cant flash them, but i read that someone there can set them up (https://github.com/rospogrigio/localtuya/issues/443)... whats different in my story?

**Comment by Scope666** [comment]
Not sure, I could never get them added at all.  I'd love to be able to though...
